### DISCUSSION TITLE
will there be a version for 1.21.1arclight ?

### DISCUSSION THREAD

**Discussion by tito323** [IzzelAliz/Arclight#1542] [Q&A] [answered]
I would like to see an arclight version for this version

**Comment by qyl27** [comment, marked as answer]
Yes, in our Actions page, FeudalKings is for 1.21.1.

**Reply by Mashndrow** [reply]
Is there a snapshot jar for it cause I can't find it

**Reply by qyl27** [reply]
> Is there a snapshot jar for it cause I can't find it

Yes, in each build, navigate to the summary, and scroll down to artifact, it is a zip file containing fabric and neoforge jars.
![image](https://github.com/user-attachments/assets/2159cbe0-7e46-4645-958c-6c0f9da38813)
